community/common/src/main/java/org/neo4j/helpers/FeatureToggles.java

```diff
@@ -17,7 +17,7 @@
  * You should have received a copy of the GNU General Public License
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
-package org.neo4j.unsafe.impl.internal.dragons;
+package org.neo4j.helpers;
```

Comment:
Does this mean it's public API now? I'd prefer to keep it non-public.

Reply:
Its really good point; thats wrong them - have moved into another package `org.neo4j.util` inside commons